### ISSUE TITLE
Does this mean you are still working on this?? 

### ISSUE THREAD

**Issue by TheCorp** [Aali132/ff7_opengl#1]
Haven't been able to get this to work and love that you are putting this on github... hoping you are still working on it 👍

**Comment by julianxhokaxhiu** [comment]
I don't think so. I'm one of the developers working behind this new fork: https://github.com/FFT-Hackers/FF7_OpenGL

Feel free to try that out.

**Comment by TheCorp** [comment]
Glad to see someone is trying to get this to work :)

**Comment by DLPB2** [comment]
> Haven't been able to get this to work and love that you are putting this on github... hoping you are still working on it 👍

There have been many forks.  Two main ones are FFNx and my mod -
https://thereunion.live/

**Comment by TheCorp** [comment]
Thank you!
